core/src/test/java/org/hibernate/ogm/backendtck/queries/CompositeIdQueriesTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.queries;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.hibernate.ogm.utils.SessionHelper.asProjectionResults;
+import static org.hibernate.ogm.utils.SessionHelper.delete;
+import static org.hibernate.ogm.utils.SessionHelper.persist;
+
+import java.util.List;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.ogm.backendtck.id.Label;
+import org.hibernate.ogm.backendtck.id.News;
+import org.hibernate.ogm.backendtck.id.NewsID;
+import org.hibernate.ogm.utils.GridDialectType;
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.SkipByGridDialect;
+import org.hibernate.ogm.utils.TestSessionFactory;
+import org.hibernate.ogm.utils.SessionHelper.ProjectionResult;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * @author Davide D'Alto
+ */
+@SkipByGridDialect(
+		value = { GridDialectType.CASSANDRA, GridDialectType.COUCHDB, GridDialectType.EHCACHE, GridDialectType.HASHMAP, GridDialectType.INFINISPAN },
+		comment = "These queries won't work with Hibernate Search because it requires a FieldBridge" )
```

Comment:
EmbeddedId can be store only using a FieldBridge, this means that we don't know how the value is stored in the index. You cannot project/filter the properties of the emebedded id since they might not exist in the index.

I've created this one: https://hibernate.atlassian.net/browse/HSEARCH-1879